src/components/search/content-highlights/data/hooks.js

```diff
@@ -52,6 +52,10 @@ export const useEnterpriseCuration = (enterpriseUUID) => {
   const [fetchError, setFetchError] = useState();
 
   useEffect(() => {
+    if (!enterpriseUUID) {
```

Comment:
If `enterpriseUUID` is not yet present, do nothing instead of calling the `enterprise_curations` API with an `undefined` as its enterprise UUID, throwing lots of `undefined is not a valid uuid` errors in enterprise catalog.